testing/features/support/pages/buttons.rb

```diff
@@ -4,9 +4,9 @@ module DesignSystem
   class Buttons < SitePrism::Page
     set_url "/iframe.html?id=3-components--buttons&viewMode=story"
 
-    element :primary, "[type='button']:not(.cads-button)"
-    element :secondary, "[type='button'].cads-button.cads-button__secondary"
-    element :tertiary, "[type='button'].cads-button.cads-button__tertiary"
+    element :primary, ".cads-button.cads-button__primary"
```

Comment:
Updated these tests because these selectors are incorrect based on my latest changes. However these seem to be doing the job of visual regression testing but by introspecting styles which seems weird?

Reply:
So these are doing some functional tests (Which backstop basically doesn't do). I've not dived heavily into backstop but these tests aren't ones we have replicated there.

The styling here is just used as locators (Backstop looks at either a large portion of a page or the page proper).